### PR TITLE
Rp4 page

### PR DESCRIPTION
**Pull request by jlpoltrack**
Title

### CODE REVIEW

**Comment by olliw42** [comment]
NICE :)

lgtm

File: docs/RADIOMASTER_RP4.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# mLRS Documentation: RadioMaster RP4 Receiver #
+
+([back to main page](../README.md))
+
+The RadioMaster RP4 is a good option for a 2.4 GHz receiver as it offers true diversity, PA & LNA, a TCXO and additional pads for RC out functionality.  Furthermore, it is also able to used as a Tx module with SiK like functionality.
```
[the remark this answers is not in the document]
Comment (jlpoltrack) [reply docs/RADIOMASTER_RP4.md:5]: Addressed

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ Documentation for the [mLRS project](https://github.com/olliw42/mLRS).
 
 ### ESP Hardware ###
 - [ELRS Receivers](docs/ELRS_RECEIVERS.md)
+- [RadioMaster RP4](docs/RADIOMASTER_RP4.md)
```
[the remark this answers is not in the document]
Comment (jlpoltrack) [reply README.md:35]: Addressed

File: docs/RADIOMASTER_RP4.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# mLRS Documentation: RadioMaster RP4 Receiver #
```
[the remark this answers is not in the document]
Comment (jlpoltrack) [reply docs/RADIOMASTER_RP4.md:1]: Addressed

**Comment by jlpoltrack** [comment]
Items should be addressed, will merge in an hour if no further feedback.

**Pull request merged**